src/examples/generation.ts

```diff
@@ -2,19 +2,18 @@ import {
     Armature,
     CostFunction,
     GeneratorInstance,
-    GeometryNode,
     Light,
     Material,
-    Matrix,
-    Model,
     Node,
     Point,
-    Quaternion,
     Renderer,
     RGBColor,
     Shape
 } from '../calder';
 
+// tslint:disable-next-line:import-name
+import Bezier = require('bezier-js');
```

Comment:
Are we going to require the user imports this library in their calder code, or could we abstract this?

Reply:
I mean, I guess it matters less since we're research focused now, and not user-focused.

Reply:
we could always provide our own wrapper, but yeah for now they do